guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<blockquote>The <code>arcgis.learn</code> module includes <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#timeseriesnetwork" target="_blank">TimeSeriesNetwork</a> class to train deep learning models on <strong>timeseries</strong> tabular data.</blockquote><blockquote><strong>Timeseries </strong>Tabular data can be any  in the form of a <a href="https://developers.arcgis.com/python/guide/working-with-feature-layers-and-features/" target="_blank">feature layer</a> or <a href="https://developers.arcgis.com/python/guide/introduction-to-the-spatially-enabled-dataframe/" target="_blank">spatially enabled dataframe</a> or a simple dataframe. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>